shakenfist/tests/test_db.py

```diff
@@ -7,6 +7,8 @@
 
 
 class DBTestCase(testtools.TestCase):
+    maxDiff = None
```

Comment:
You variable name weird.

Reply:
LOL. Not me bro. That's configuring testtools to output the full diff when an expect fails.

Reply:
Oh nice.